### DISCUSSION TITLE
Evaluation warning on renamed options, new name doesn't work.

### DISCUSSION THREAD

**Discussion by pol-epsevg** [nix-community/nixvim#2428] [Q&A]
I'm quite new to Nix, so I might be doing something wrong here.
I've seen something along the lines of "Evaluation warning: foo has been renamed to bar", and normally you just change foo for bar and it goes away as you'd expect. But with nixvim warnings:

```
trace: evaluation warning: The option `plugins.bufferline.separatorStyle` defined in `/nix/store/.../plugins.nix` has been renamed to `plugins.bufferline.settings.options.separator_style`.
```
But when I change `plugins.bufferline.separatorStyle` to `plugins.bufferline.settings.options.separator_style`, it says it can't find the option.

Similar thing happens with this warning:
```
trace: evaluation warning: Nixvim: `plugins.web-devicons` was enabled automatically because the following plugins are enabled.
This behaviour is deprecated. Please explicitly define `plugins.web-devicons.enable` or alternatively
enable `plugins.mini.enable` with `plugins.mini.modules.icons` and `plugins.mini.mockDevIcons`.
plugins.nvim-tree
plugins.bufferline
```
If I enable web-devicons it tells me it can't find the option. What am I doing wrong?

**Comment by MattSturgeon** [comment]
It'd be helpful to see your actual config to understand what's going on here. Do you have it in a public repo?

Or alternatively could you upload the relevant modules and flake.nix files somewhere (GitHub Gists?) and link to them here?

Additionally, pasting the actual error printed when you get the option doesn't exist error would be helpful. There might be some hints in the error message that makes this easier for us to figure out.

My initial guess is that you either need a `programs.nixvim` prefix or vice versa, but without some more context I'm taking wild stabs in the dark 😁

**Comment by pol-epsevg** [comment]
Honestly, I don't know how I've been able to consistently mess it up somehow without noticing. 

I tried to reproduce the actual errors, but I noticed I made a typo. When I fixed it, it built correctly. This must mean the several other times I've tried, I've made some similar mistake
